src/debian/90-sysctl-hardening.conf

```diff
@@ -6,7 +6,7 @@ dev.tty.ldisc_autoload = 0
 
 # Several FIFO / symlink / hardlink protections to aleviate /tmp based
 # attacks or TOCTOU using "other" writable folders.
-fs.protected_fifos = 1
+fs.protected_fifos = 2
```

Comment:
Misleading since this you found that these settings are overriden. It would be best to just remove them here.